core/src/main/java/org/bitcoinj/wallet/DeterministicKeyChain.java

```diff
@@ -711,10 +711,10 @@ public int numLeafKeysIssued() {
 
     @Override
     public Instant getEarliestKeyCreationTimeInstant() {
-        if (seed != null)
-            return Instant.ofEpochSecond(seed.getCreationTimeSeconds());
-        else
-            return Instant.ofEpochSecond(getWatchingKey().getCreationTimeSeconds());
+        return Instant.ofEpochSecond(seed != null ?
+                seed.getCreationTimeSeconds() :
+                getWatchingKey().getCreationTimeSeconds()
+        );
```

Comment:
I would probably do the white-space differently ( `?` and `:` as first character of line) but that's minor.